packages/net/http/_request_builder.pony

```diff
@@ -57,3 +57,6 @@ class _RequestBuilder is TCPConnectionNotify
       end
     end
     true
+
+  fun ref connect_failed(conn: TCPConnection ref) =>
+    None
```

Comment:
This comment is also still outstanding - some form of error handling should be done here.